nipype/interfaces/ants/registration.py

```diff
@@ -1034,3 +1034,115 @@ def _list_outputs(self):
         if len(self.inputs.save_state):
             outputs['save_state'] = os.path.abspath(self.inputs.save_state)
         return outputs
+
+
+class MeasureImageSimilarityInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(
+         2, 3, 4,
+         argstr='--dimensionality %d', usedefault=False,
+         position=1, desc='Dimensionality of the fixed/moving image pair',
+         )
+    fixed_image = File(
+         exists=True, mandatory=True,
+         desc=('image to which the moving image is warped'),
+         )
+    moving_image = File(
+         exists=True,mandatory=True,
+         desc=('image to apply transformation to (generally a coregistered functional)'),
+         )
+    metric = traits.Enum(
+         "CC", "MI", "Mattes", "MeanSquares", "Demons", "GC", 
+         argstr="%s", mandatory=True,
+         )
+    metric_weight = traits.Float(requires=['metric'], desc='',mandatory=True,) 
+    radius_or_number_of_bins = traits.Int(requires=['metric'], desc='',mandatory=True,)
+    sampling_strategy = traits.Enum(
+         "None", "Regular", "Random", None,
+         requires=['metric'],mandatory=True,
+         )
+    sampling_percentage = traits.Either(
+         traits.Range(low=0.0, high=1.0), None,
+         requires=['metric'], mandatory=True,
+         )
+    fixed_image_mask = File(
+        exists=True, argstr='%s',
+        desc='mask used to limit metric sampling region of the fixed image',
+        )
+    moving_image_mask = File(
+        exists=True, requires=['fixed_image_mask'],
+        desc='mask used to limit metric sampling region of the moving image',
+        )
+
+
+class MeasureImageSimilarityOutputSpec(TraitedSpec):
+    similarity =traits.Float() 
+
+
+class MeasureImageSimilarity(ANTSCommand):
+
+    """
```

Comment:
No newline before docstring. Can you add a basic description of `MeasureImageSimilarity`? [e.g.](https://github.com/stnava/ANTs/blob/c7ba17797cf51683376886ac6c2179b572cee98c/Examples/MeasureImageSimilarity.cxx#L536-L537):

"Calculate the similarity between two images using various metrics."